lib/nsxv.py

```diff
@@ -353,6 +374,18 @@ def __str__(self):
     ret_str.extend(stripped_ret_lines)
     return ''.join(ret_str)
 
+  def _DropMismatchIpAddress(self, ipaddress, direction_address, af):
+    modified_ipaddress = []
+    for addr in ipaddress:
+      if ((af == 4 and not type(addr) is nacaddr.IPv4)
```

Comment:
Do not compare with type, use isinstance.
https://stackoverflow.com/questions/30049410/why-use-isinstance-instead-of-type